_posts/2021-09-16-data-prepper-roadmap.markdown

```diff
@@ -8,45 +8,46 @@ date:   2021-09-16 01:01:01 -0700
+If you want to add something that is not in the [public roadmap](https://github.com/opensearch-project/data-prepper/projects/1) for OpenSearch Data Prepper, that’s a perfect opportunity to contribute! We are looking for contributors to help develop new processors and new source plugins and accelerate development of processors and source plugins on the roadmap. This public roadmap follows the same principles in the blog post outlining the [OpenSearch public roadmap](https://opensearch.org/blog/update/2021/05/opensearch-roadmap-announcement/). It is worth emphasizing a few points:
 
 1. Date and milestones reflect intentions rather than firm commitments. Dates may change as anyone learns more or encounters unexpected issues. The roadmap will help make changes more transparent so that anyone can plan around them accordingly. 
-2. You can create a [feature request](https://github.com/opensearch-project/.github/blob/main/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md) in the relevant GitHub repo for the feature and socialize the request. A maintainer or someone else in the community may pick this feature up and work on it. As progress is made the maintainers of the repo will help get the feature onto the roadmap.
-3. Another option is to build the feature yourself. To do this create a proposal as a GitHub issue in the relevant repo and use the [proposal template](https://github.com/opensearch-project/.github/blob/main/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md). Offer your commitment to build it. The maintainers of the repo will work with you to figure out how best to proceed. 
-
-Please also see [OpenSearch FAQ 1.19](https://opensearch.org/faq#q1.19) for more details. Learn more about Data Prepper by accessing the code, examples, and documentation in the [Data Prepper GitHub repository](https://github.com/opensearch-project/data-prepper). You can also connect with us and provide feedback in our [forum category](https://discuss.opendistrocommunity.dev/t/about-the-data-prepper-category/7038).
\ No newline at end of file
+2. You can create a [feature request](https://github.com/opensearch-project/.github/blob/main/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md) in the relevant GitHub repo for the feature and socialize the request. A maintainer or someone else in the community may pick up this feature and work on it. As progress is made, the maintainers will help get the feature onto the roadmap.
+3. You can build a feature yourself. To do this create a proposal as a GitHub issue in the relevant repo and use the [proposal template](https://github.com/opensearch-project/.github/blob/main/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md). Offer your commitment to build it. The maintainers will work with you to figure out how best to proceed. See [OpenSearch FAQ 1.19](https://opensearch.org/faq#q1.19) for more details. 
+ 
```

Comment:
It might not be a bad idea to let readers know about finding "good first issues" if they want to contribute.

Perhaps we can add a new point.

> 4. Would you like to contribute? Maybe you don't know where to start. The Data Prepper maintainers curate [good first issues](https://github.com/opensearch-project/data-prepper/labels/good%20first%20issue). These are issues that we believe new contributors could take on. Please comment on the issue if you would like to work on it.